src/include/sof/audio/component.h

```diff
@@ -251,10 +251,11 @@ enum {
 		  (uint32_t)((pcd)->cpu_delta_peak))
 
 #define comp_perf_avg_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u",\
+	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u %u",\
 		  (uint32_t)((comp_p)->frames),            \
 		  (uint32_t)((comp_p)->period),			    \
 		  (uint32_t)((pcd)->cpu_delta_sum),			\
+		  (uint32_t)((pcd)->peak_mcps_cnt),			\
```

Comment:
let's move this peak_mcps_cnt to the end if you really want to print it. with an analysis tool, this looks not very necessary.

Reply:
Seppo's script is based on this, :).
move to end is reasonable, it will not break your current script.